mix/mix_protocol.nim

```diff
@@ -57,6 +57,12 @@ proc cryptoRandomInt(max: int): Result[int, string] =
 proc toUnixNs(t: Time): int64 =
   t.toUnix().int64 * 1_000_000_000 + times.nanosecond(t).int64
 
+func byteToHex(b: byte): string = 
```

Comment:
Are these functions used anywhere?

Reply:
They were used when we were outputting the message in hex form, but not any more. will drop them.